src/commands/list-kinesis-streams.js

```diff
@@ -0,0 +1,227 @@
+const _ = require("lodash");
+const AWS = require("aws-sdk");
+const Table = require("cli-table");
+const { Command, flags } = require("@oclif/command");
+const { checkVersion } = require("../lib/version-check");
+const Lambda = require("../lib/lambda");
+require("colors");
+const uuid = require("uuid");
+
+const ONE_MIN_IN_SECONDS = 60;
+const MAX_RECORDS_PER_SHARD = 1000;
+const BYTES_INCOMING_PER_SHARD = 1024 * 1024;
+const BYTES_OUTGOING_PER_SHARD = 2048 * 1024;
+
+const formatFloat = num => {
+	return Number.parseFloat(num).toFixed(3);
+};
+
+class ListKinesisStreamsCommand extends Command {
+	async run() {
+		const { flags } = this.parse(ListKinesisStreamsCommand);
+		const { region, profile } = flags;
+
+		AWS.config.region = region;
+		if (profile) {
+			AWS.config.credentials = new AWS.SharedIniFileCredentials({ profile });
```

Comment:
there's a `lib/aws` module, which saves you some hassle for this, see https://github.com/lumigo-io/lumigo-CLI/blob/master/src/commands/list-lambda.js for how I'm using it in other places

Reply:
Done